frontend/packages/kubevirt-plugin/src/plugin.ts

```diff
@@ -50,6 +53,21 @@ const plugin: Plugin<ConsumedExtensions> = [
       mergeAfter: 'Pods',
     },
   },
+  {
+    // NOTE(yaacov): vmtemplates is a template resource with a selector.
+    // 'NavItem/ResourceNS' is used, and not 'NavItem/Href', because it injects
+    // the namespace needed to get the correct link to a resource ( template with selector ) in our case.
```

Comment:
In theory, `NavItem/Href` should only be used for URLs outside of Console :smile:

So using `NavItem/ResourceNS` pointing to VM template resource is self-explanatory, I think.

Reply:
This comment is here because of https://github.com/openshift/console/pull/1754#discussion_r295429625 

May be an over kill :smile: I can remove it if you go not think someone may refactor it into an href in the future :tulip: